hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: minor
+
+This release adds support for Django 5.0, and drops support for Django < 4.2.
```

Comment:
I think it's more accurate to describe this as _improving_ support, since it was only new-in-5.0 features that didn't work (contrasting cases where the update itself broke previously-working code)

```suggestion
This release improves support for Django 5.0, and drops support for end-of-life Django versions (< 4.2).
```